### PR TITLE
Add installation instructions for all platforms

### LINKED ISSUE #4511: Make the linux installation experience better
We're missing any sort of Linux installation instructions, the overall experience isn't polished yet on that platform. On fresh Ubuntu installs I had to install these packages to run the appimage:
```
sudo apt install zlib1g-dev libfuse2 libnss3 libatk1.0-0 libatk-bridge2.0-0 libcups2 libgtk-3-0 libgbm-dev libasound2 xdg-utils
```

### PR DESCRIPTION
**Pull request by pierremtb**
Fixes #4511

For now this is a new INSTALL.md file, we need to see if we should get it pulled and displayed by the download page on the website. Or at least just point to it

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+2. Install the dependencies needed to run the [AppImage format](https://appimage.org/).
+    -  On Ubuntu, install the FUSE library with these commands in a terminal.
+       ```bash
+       sudo apt update
+       sudo apt install libfuse2
+       ```
+    - Further desktop environment can be achieved by installing `appimaged`, see [these steps](https://github.com/probonopd/go-appimage/blob/master/src/appimaged/README.md#initial-setup). 
+
+2. Once downloaded, copy the downloaded `Zoo Modeling App-{version}-{arch}-linux.AppImage` to the directory of your choice, for instance `~/Applications`.
```
Comment (TomPridham) [inline INSTALL.md:34]:
```suggestion
    - Follow the [these steps](https://github.com/probonopd/go-appimage/blob/master/src/appimaged/README.md#initial-setup) to install `appimaged`. It is a daemon that makes interacting with AppImage files more seamless. 

2. Once the dependencies are installed, copy the downloaded `Zoo Modeling App-{version}-{arch}-linux.AppImage` to the directory of your choice, for instance `~/Applications`.
```

Comment (pierremtb) [reply INSTALL.md:34]: Appreciate it, thank you @TomPridham!

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Setting Up Zoo Modeling App
+
+Compared to other CAD software, getting Zoo Modeling App up and running is quick and straightforward across platforms. It's about 100MB to download and is quick to install.
```
Comment (franknoirot) [inline INSTALL.md:3]: Okay _marketing_ 😄

**Review by franknoirot — APPROVED** [review]
Well written!

**Pull request merged**